src/api/java/io/github/cvc5/Term.java

```diff
@@ -155,8 +158,20 @@ public Term substitute(Term term, Term replacement)
   private native long substitute(long pointer, long termPointer, long replacementPointer);
 
   /**
-   * @return the result of simultaneously replacing 'terms' by 'replacements'
-   * in this term
+   * Simultaneously replace {@code terms} with {@code replacements} in this
+   * term.
+   *
+   * In the case that terms contains duplicates, the replacement earliest in
+   * the vector takes priority. For example, calling substitute on
+   * {@code f(x,y)} with {@code terms = { x, z }},
+   * {@code replacements = { g(z), w }} results in the term
+   * {@code f(g(z),y)}.
+   *
+   * @api.note This replacement is applied during a pre-order traversal and
+   *           only once (it is not run until fixed point).
```

Comment:
only once to the sort in cvc5.h should be changed as well to 
only once since we have multiple sorts here.